Ports/.port_include.sh

```diff
@@ -422,7 +422,8 @@ clean() {
 }
 clean_dist() {
     for f in "${files[@]}"; do
-        read url filename hash <<< $(echo "$f")
+        read -r url hash <<<"$f"
```

Comment:
```suggestion
        read -r url hash <<< "$f"
```

Also, do we really want `-r` here? A backslash shouldn't be in here ever either way, so I'd say to omit `-r` for consistency with `fetch`.

Reply:
Done!